enterprise/backend/src/metabase_enterprise/advanced_config/file.clj

```diff
@@ -219,7 +219,11 @@
       (str/join parts))))
 
 (defn- expand-templates-in-str [s]
-  (str/join (map expand-template-str-part (params.parse/parse s))))
+  (if (and (str/starts-with? s "{{{") (str/ends-with? s "}}}"))
+    (-> s
+        (subs 3 (- (count s) 3))
+        str/trim)
+    (str/join (map expand-template-str-part (params.parse/parse s)))))
```

Comment:
This works, so not a blocking comment — but might be slightly more straightforward with `re-matches`? i.e.
```suggestion
  (if-let [[_, raw-string] (re-matches #"\{\{\{(.+)\}\}\}" s)]
    (str/trim raw-string)
    (str/join (map expand-template-str-part (params.parse/parse s))))
```

Reply:
Thanks. I did your suggestion